docs/reference/commandline/search.md

```diff
@@ -97,10 +97,10 @@ be in the range between 1 and 100. The default value of `--limit` is 25.
 
 ### Filtering
 
-The filtering flag (`-f` or `--filter`) format is a `key=value` pair. If there is more
-than one filter, then pass multiple flags (e.g. `--filter "foo=bar" --filter "bif=baz"`)
+The filtering flag (`-f` or `--filter`) format is a `key=value` pair. The supported key and value given below. If there is more
+than one filter, then pass multiple flags (e.g. `--filter is-automated=true --filter stars=3`)
 
-The currently supported filters are:
+The currently supported filters(key=value) are:
```

Comment:
The paragraph above already describes that the filters are `key=value` pairs, so I don't think we need to repeat that here;

```suggestion
The currently supported filters are:
```